cli/tools/lint/mod.rs

```diff
@@ -289,10 +290,14 @@ impl WorkspaceLinter {
     let exclude = lint_options.rules.exclude.clone();
 
     let plugin_specifiers = lint_options.plugins.clone();
-    let lint_rules = self.lint_rule_provider.resolve_lint_rules_err_empty(
+    let lint_rules = self.lint_rule_provider.resolve_lint_rules(
       lint_options.rules,
       member_dir.maybe_deno_json().map(|c| c.as_ref()),
-    )?;
+    );
+    if lint_rules.rules.is_empty() && plugin_specifiers.is_empty() {
+      bail!("No rules have been configured")
+    }
```

Comment:
I think we should do that on line 340 after the plugins are actually resolved, but maybe it's an edge case not worth exploring

Reply:
Why? Wouldn't no plugins be resolved?

Reply:
Oh, is it possible for `create_runner_and_load_plugins` to not return any plugins if there's specifiers for some reason?

Reply:
You're right, the worst case is that plugin can't be loaded but that would result in an error. So disregard